rfcs/detecting_sensitive_textual_content/20230309-implementation_plan_sensitive_terms_list.md

```diff
@@ -0,0 +1,158 @@
+# Implementation Plan: Managing A Sensitive Terms List
+
+> **Content Warning**
+>
+> This document **does not** include sensitive content or direct links to
+> sensitive content. It _does_ discuss different sub-categorizations of
+> sensitive content at times.
+>
+> This document should not be considered sensitive but please view at your own
+> discretion, regardless.
+
+## Reviewers
+
+<!-- Choose two people at your discretion who make sense to review this based on their existing expertise. Check in to make sure folks aren't currently reviewing more than one other proposal or RFC. -->
+
+- [ ] @sarayourfriend
+- [ ] @dhruvkb
+
+The Openverse project
+[Filter and blur sensitive results by term matching #377](https://github.com/orgs/WordPress/projects/70?query=is%3Aopen+sort%3Aupdated-desc&pane=issue&itemId=19530115)
+requires that we compare the textual contents of Openverse media against a
+denylist of sensitive terms. We use this list to identify media that should be
```

Comment:
Just a nit that "denylist" might imply that we _are_ rejecting these documents (despite the qualification in the following sentence).

Reply:
Good point. Switched to "list" in 950bc718.